index.html

```diff
@@ -5,6 +5,11 @@
 <img src="images/workshop-large.jpg" class="hero" alt="people working together on computers"></img>
 <div class="container">
     <h1 class="headline">We build bridges.</h1>
+    <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+          Empower people with technology through teaching and facilitating access, enlarging the community of people who give back and teach others.
```

Comment:
@mfjenn  Hi Mary, here is what I am looking at, we need this to transition well into the sentence.